lib/src/base-styles/_nx-colors.scss

```diff
@@ -170,6 +170,11 @@
   --nx-color-small-threat-counter-zero-background: var(--nx-color-interactive-background-disabled);
   --nx-color-small-threat-counter-zero-text: var(--nx-swatch-grey-40);
 
+  //NxSmallVulnerabilityCounter
+  --nx-color-small-vulnerability-counter-zero-background: var(--nx-swatch-grey-80);
+  --nx-color-small-vulnerability-counter-zero-border: var(--nx-swatch-grey-10);
+  --nx-color-small-vulnerability-counter-zero-text: var(--nx-swatch-grey-05);
```

Comment:
The dark mode styles for the zero counter are incorrect.  This block presumably needs to be repeated and updated in the dark mode section.  Either that or this component should use the `--nx-color-small-threat-counter-zero-…` variables.

Reply:
> The dark mode styles for the zero counter are incorrect relative to the latest mockups. This block presumably needs to be repeated and updated in the dark mode section. Either that or this component should use the `--nx-color-small-threat-counter-zero-…` variables.

Updated dark mode variables here: [a229cf8](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/a229cf8b50783097ffda6ceb79a01b03dd95022a)

Will update visual tests once the whole PR is approved.